lib-integration-java/src/main/java/com/digitalasset/integration/internal/codec/strategies/DomXsdEncodeStrategy.java

```diff
@@ -243,12 +243,8 @@ private void applyCardinality(
             }
         } else if(cardinality.equals(FieldMetadata.Cardinality.MANY1)) {
             List<Value> values = value.asList().get().getValues();
-            if (values.isEmpty()) {
-                throw new CodecException("Unexpected empty list for type "+elemType.toString());
-            } else {
-                for (Value elemValue : values) {
-                    encoder.accept(Tuple.of(elemType, elemValue));
-                }
+            for (Value elemValue : values) {
+                encoder.accept(Tuple.of(elemType, elemValue));
```

Comment:
we encountered an XSD in the wild that has XML elements with nothing inside it. Removing this restriction was necessary to get things working.